vignettes/cloud.Rmd

```diff
@@ -0,0 +1,74 @@
+---
+title: "cloud"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{cloud}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(revdepcheck)
+```
+
+# Running reverse dependencies in the cloud
+
+revdepcheck now supports running reverse dependencies in the AWS cloud, via a suite a functions all prefixed with `cloud_*()`.
```

Comment:
Add a sentence mentioning that this is currently available only to RStudio employees, but we'll work to make it available to the wider community in the future.